lib/sensu-plugins-aws/cloudwatch-common.rb

```diff
@@ -1,8 +1,11 @@
 require 'aws-sdk'
 
 module CloudwatchCommon
+
+  include Common
+
   def client
-    Aws::CloudWatch::Client.new
+    Aws::CloudWatch::Client.new aws_config
```

Comment:
Ok, also this then https://github.com/sensu-plugins/sensu-plugins-aws/blob/master/bin/check-cloudwatch-alarm.rb#L80 ?